scalafmt-macros/shared/src/main/scala/org/scalafmt/config/DialectMacro.scala

```diff
@@ -0,0 +1,34 @@
+package org.scalafmt.config
```

Comment:
@tgodzik non-blocking comment for this pr: should we consider copying this functionality into scalameta? i know you have been working on making that working with scala3, perhaps there's a way to have two macro-based implementations, one for scala2 and another for scala3.